Vanilla-JS-Projects/Basic/Joke-Telling-Website/README.md

```diff
@@ -0,0 +1,80 @@
+<h1 align='center'><b>💥 Joke Telling Website 💥</b></h1>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>HTML, CSS, Vanilla-JS 🎮</h3>
```

Comment:
remove this line.

Reply:
<h3 align='center'>HTML, CSS, Vanilla-JS 🎮</h3>

or
<!-- -------------------------------------------------------------------------------------------------------------- -->

Reply:
---- which line

Reply:
h3 heading line

Reply:
go through other projects directory and take reference ok

Reply:
check the other projects readme file how the tech stacks are written ok